src/coreclr/gc/unix/config.gc.h.in

```diff
@@ -17,6 +17,7 @@
 #cmakedefine01 HAVE_SYSCTLBYNAME
 #cmakedefine01 HAVE_PTHREAD_CONDATTR_SETCLOCK
 #cmakedefine01 HAVE_CLOCK_GETTIME_NSEC_NP
+#cmakedefine01 HAVE_CLOCK_MONOTONIC
```

Comment:
I checked that it's [covered in the cross-compiling section already](https://github.com/dotnet/runtime/blob/22a8804be314653a69e4e052044426e9837b80ff/eng/native/tryrun.cmake#L33) 👍